CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTSystemTemplateActionHandler.m

```diff
@@ -0,0 +1,30 @@
+//
+//  CTSystemTemplateActionHandler.m
+//  CleverTapSDK
+//
+//  Created by Nishant Kumar on 27/03/25.
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "CTSystemTemplateActionHandler.h"
+#import "CTConstants.h"
+
+@implementation CTSystemTemplateActionHandler
+
+#pragma mark Push Permission System App Function
+
+- (void)setPushPrimerManager:(CTPushPrimerManager *)pushPrimerManagerObj {
+    pushPrimerManager = pushPrimerManagerObj;
+}
+
+- (void)promptPushPermission:(BOOL)fbSettings {
```

Comment:
This method implementation differs from the `[CleverTap promptForPushPermission:]`. Is this intended and why the difference?

Reply:
This is intended, now we are using push status so returned early.